config/logstash.yml

```diff
@@ -113,3 +113,12 @@
 #
 # Where to find custom plugins
 # path.plugins: []
+#
+# ------------ Slow log thresholds ---------
+#
+# An slow log threshold is a time limit for an operation that when is overcome it will
+# be reported at the related log file. Time is seconds.
+#
+# The last building block of the component is the log level (default: warn)
+#
+# filters.dns.getaddress.warn: 5
```

Comment:
I've written this as a kind of example, but it fits the format I think is appropriate, I would like to get in a namespace that makes the user able to locate which operation, direct or logical groups is invoked.

With `filters.dns.[operation_label].[log_level]` all the necessary info to get the context of the threshold is there, with your naming proposal the user can not know the context where this threshold is defined. Also what happen when there are two thresholds for a plugin?